src/components/UserProfile.jsx

```diff
@@ -35,6 +36,10 @@ const UserProfile = () => {
               <ListItem.Title>{item.track}</ListItem.Title>
               <ListItem.Subtitle>{item.artists}</ListItem.Subtitle>
             </ListItem.Content>
+            <Button 
+              testID={`delete-button-${item.id}`}
+              onPress={() => {PostService.delete(item.id, credentials)}}
+            >Delete</Button>
```

Comment:
just a note to say we can fix this in styling later

Reply:
yep :)